adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/events/TicketsEvent.java

```diff
@@ -2,9 +2,20 @@
 
 import java.util.List;
 
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.microservicesteam.adele.messaging.events.Event;
 import com.microservicesteam.adele.ticketmaster.model.Position;
 
-interface TicketsEvent extends Event {
+@JsonTypeInfo(use=JsonTypeInfo.Id.NAME,
+        include=JsonTypeInfo.As.PROPERTY,
+        property="type")
```

Comment:
As I can see, @kelemena wanted to avoid to have the immutable concrete class name as type. If I comment out `@JsonTypeName`, it serialize as:
```json
{"type":"ImmutableTicketsCreated","positions":[{"eventId":1,"sectorId":1,"id":1}]}
```
The `@class` value seems wierd for me, because it results in:
```json
{"@class":"ImmutableTicketsCreated","positions":[{"eventId":1,"sectorId":1,"id":1}]}
```